test/semantics/omp-clause-validity01.f90

```diff
@@ -445,7 +445,6 @@
   enddo
   !$omp end taskloop simd
 
-  !ERROR: REDUCTION clause is not allowed on the TASKLOOP SIMD directive
   !$omp taskloop simd reduction(+:a)
```

Comment:
No reduction clause can be specified. (4.5 Spec P92)

Reply:
Interesting. The reduction clause actually _can_ be specified in 5.0, but not in 4.5. Which is our "source of truth" here? I assume 4.5, in which case I'll add this limitation back in.